### ISSUE TITLE
About training data

### ISSUE THREAD

**Issue by ctwayen** [Deep-MI/FastSurfer#47]
Hi. Just read ur guy's paper. Amazing work! However, one thing that I found confusing is that what data did you guys used as ground truth to calculate loss? Did you pass all training samples into freesuffer and use the results as ground truth? Thank you in advance!

**Comment by LeHenschel** [comment]
Hey,

thank you :). Yes, that is exactly what we did. All datasets were processed using FreeSurfer v6.0 and we used the resulting segmentation following the ‘‘Desikan–Killiany–Tourville’’ (DKT) protocol atlas as ground truth (aparc.DKTatlas+aseg.mgz).

Best,
Leonie

**Comment by ctwayen** [comment]
Hi,

Thanks for the reply! I really appreciate it!

Best,
Xinrui
